diff -auurN bash-4.3.orig/config-top.h bash-4.3/config-top.h
--- bash-4.3.orig/config-top.h	2014-02-27 12:03:46.939523979 +0100
+++ bash-4.3/config-top.h	2014-02-27 12:08:17.813306605 +0100
@@ -59,14 +59,14 @@
 /* The default value of the PATH variable. */
 #ifndef DEFAULT_PATH_VALUE
 #define DEFAULT_PATH_VALUE \
-  "/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/bin:/sbin:."
+  "/usr/local/bin:/usr/bin"
 #endif
 
 /* The value for PATH when invoking `command -p'.  This is only used when
    the Posix.2 confstr () function, or CS_PATH define are not present. */
 #ifndef STANDARD_UTILS_PATH
 #define STANDARD_UTILS_PATH \
-  "/bin:/usr/bin:/sbin:/usr/sbin:/etc:/usr/etc"
+  "/usr/bin:/usr/sbin"
 #endif
 
 /* Default primary and secondary prompt strings. */
@@ -80,7 +80,7 @@
 #define KSH_COMPATIBLE_SELECT
 
 /* System-wide .bashrc file for interactive shells. */
-/* #define SYS_BASHRC "/etc/bash.bashrc" */
+#define SYS_BASHRC "/etc/bashrc"
 
 /* System-wide .bash_logout for login shells. */
 /* #define SYS_BASH_LOGOUT "/etc/bash.bash_logout" */
diff -auurN bash-4.3.orig/config.h.in bash-4.3/config.h.in
--- bash-4.3.orig/config.h.in	2014-02-27 12:03:46.939523979 +0100
+++ bash-4.3/config.h.in	2014-02-27 12:05:12.509569496 +0100
@@ -229,7 +229,7 @@
 
 /* System paths */
 
-#define DEFAULT_MAIL_DIRECTORY "/usr/spool/mail"
+#define DEFAULT_MAIL_DIRECTORY "/var/mail"
 
 /* Characteristics of the system's header files and libraries that affect
    the compilation environment. */
